applications/app/views/signup/newsletterContent.scala.html

```diff
@@ -91,11 +91,25 @@ <h3 class="signup-confirmation--success">Check your Inbox!</h3>
                     </div>
                 </a>
             </div>
-            @* This iframe is used for submitting the form without reloading the page *@
-            <iframe class="u-h" width="0" height="0" border="0" name="newsletterSignup" id="newsletterSignup" tabindex="-1"></iframe>
-            @if(EmailSignupRecaptcha.isSwitchedOn) {
+            @if(NewslettersRemoveConfirmationStep.isSwitchedOn) {
+                <div class="newsletters-heading__divider"></div>
+                <div class="newsletters-privacy-notice__container">
+                    <div class="newsletters-privacy-notice__heading">
+                        Privacy Notice
+                    </div>
+                    <div class="newsletters-privacy-notice__content">
+                        @if(EmailSignupRecaptcha.isSwitchedOn) {
```

Comment:
It looks like if both `EmailSignupRecaptcha.isSwitchedOn = True` and `NewslettersRemoveConfirmationStep.isSwitchedOn = True`, the recaptcha terms will show up twice - here and on L109

Reply:
it shouldnt there  is an else

Reply:
oh yes! sorry!!